contracts/0.8.9/StakingRouter.sol

```diff
@@ -182,6 +187,68 @@ contract StakingRouter is IStakingRouter, AccessControlEnumerable, BeaconChainDe
         emit StakingModuleFeesSet(_stakingModuleId, _treasuryFee, _moduleFee);
     }
 
+    function reportRewardsMinted(uint256[] calldata _stakingModuleIds, uint256[] calldata _totalShares)
+        external
+        onlyRole(REPORT_REWARDS_MINTED_ROLE)
+    {
+        for (uint256 i = 0; i < _stakingModuleIds.length; ) {
+            address moduleAddr = _getStakingModuleById(_stakingModuleIds[i]).stakingModuleAddress;
+            IStakingModule(moduleAddr).handleRewardsMinted(_totalShares[i]);
+            unchecked { ++i; }
+        }
+    }
+
+    function updateExitedKeysCountByStakingModule(
```

Comment:
TODO: add a handle that allows the DAO to forcibly decrease exited keys count for staking module and its operators so that it's possible to correct an oracle error manually.